test/11_dependency_versions/spam.c

```diff
@@ -0,0 +1,48 @@
+#include <Python.h>
```

Comment:
Minor detail/edit, but would there be a way of sharing all the `spam.c` versions that are the same? This would (maybe) make it more clear that they are the same, and highlight when there's something in there that is important?

Reply:
symlinks?

Reply:
Hmmm, if they work well with git & Windows?

Reply:
other options. move it to utils. Add proper variable which contains path to spam.c and copy it on test begin? 

```python 
import os
from shutil import copyfile

import utils

project_dir = os.path.dirname(__file__)

copyfile(utils.base_spam, os.path.join(project_dir, "spam.c")
```

Reply:
Hmmm, that's already slightly better. But maybe that's too complex to be worth it. Let's see if @joerick thinks this is a problem, first. It was just a thought I had when reading the changes and didn't immediately know whether this was the normal 'spam.c' or a modified version.

Reply:
Funny you should mention it...! This has been on my mind, too. I've been playing around with it on a branch for a while - just pushed it to PR #277 - curious what you think!

Reply:
Oh, wow! I'll get a look somewhere over the next few days!